pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -86,15 +84,11 @@ func (am *autoScalerManager) Sync(tac *v1alpha1.TidbClusterAutoScaler) error {
 		}
 		return err
 	}
-	permitted, err := am.checkAutoScalerRef(tc, tac)
-	if err != nil {
-		klog.Error(err)
+
+	defaultTAC(tac, tc)
+	if err := validateTAC(tac); err != nil {
 		return err
```

Comment:
Do not return error here, just print the error log and return nil, if we return err here, this CR will be synced again very soon.